lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -6,21 +6,22 @@ class Relationship
         # {http://jsonapi.org/format/#document-links Document Links}
         # {http://jsonapi.org/format/#document-resource-object-linkage Document Resource Relationship Linkage}
         # {http://jsonapi.org/format/#document-meta Docment Meta}
-        def initialize(parent_serializer, serializer, options = {}, links = {}, meta = nil)
+        def initialize(parent_serializer, serializer, serializable_resource_options, args = {})
```

Comment:
also, I see you have `args` here - what's the need for that? as apposed to named arguments with defaults?